gatorgrade/main.py

```diff
@@ -0,0 +1,38 @@
+"""Use Typer to run gatorgrade to run the checks and generate the yml file."""
+from pathlib import Path
+import typer
+
+app = typer.Typer(add_completion=False)
+FILE = "gatorgrade.yml"
+
+
+@app.callback(invoke_without_command=True)
+def gatorgrade(
+    ctx: typer.Context,
+    filename: Path = typer.Option(
+        "FILE", "--config", "-c", help="Name of the yml file."
```

Comment:
@Michionlion I had a quick question about this. So Rian fixed this line, but now it is causing errors in the code. Specifically, now that I fixed this all of the filename declaration information has to be on one line in order to follow formatting rules. This causes line 12 of main.py to be 9 characters over the 79 character limit. I have tried splitting it up onto different lines by splitting it by the different parameters for filename as well as by the parenthesis, but both of these don't follow formatting rules. Do you have any suggestions on how to fix this?

Reply:
Run `black` -- if there are conflicting formatting after `black` is run, then the configuration of your tooling should probably be adjusted.

Reply:
Okay, I did some investigating and it looks like there's some instructions on what you should do here: https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html

You should be trying to keep configuration to just the `pyproject.toml` file as much as possible, so I think this may be a good reason to get rid of `flake8` entirely; pylint does do a lot of what flake8 does, so it wouldn't be a great loss. That would enable you to bypass this issue entirely, since the default configuration for pylint conforms line-length to 88 characters, which is what `black` enforces.